python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -498,8 +523,14 @@ async def poll_loop(self):
                     self.log.critical(f"Unhandled state: {self.summary_state}")
 
                 await asyncio.sleep(1)
```

Comment:
Consider using `self.heartbeat_interval` (it is a class attribute) instead of `1` in the sleep.